web/styles/subscriptions.css

```diff
@@ -560,6 +570,20 @@ h4.user_group_setting_subsection_title {
     width: 100%;
 }
 
+.left .fa-ban,
+.right .fa-ban {
```

Comment:
Would be good to make this selector more specific to avoid these styles being applied to other places where `fa-ban` icon is used.